client-session/src/main/resources/META-INF/maven/archetype.xml

```diff
@@ -0,0 +1,9 @@
+<archetype>
+  <id>client-session</id>
+  <sources>
+    <source>src/main/java/App.java</source>
+  </sources>
+  <testSources>
+    <source>src/test/java/AppTest.java</source>
+  </testSources>
+</archetype>
```

Comment:
I'm not sure we need the archetype. I propose we remove this file.

Reply:
Removed.